web/pdf_viewer.js

```diff
@@ -2031,6 +2035,12 @@ class PDFViewer {
     this._setScale(newScale, options);
   }
 
+  rescale(scaleFactor, options) {
+    const newScaleValue =
+      Math.round(this._currentScale * scaleFactor * 100) / 100;
+    this._setScale(newScaleValue, options);
+  }
+
```

Comment:
Sorry to say, but I'm not particularly "liking" this new method. Note how changes in this file basically introduces new official viewer-APIs, and I'm not sure if this method is something we want to expose. (Also, there's a bunch of "required" validation currently missing here.)

Let's first address my question above, regarding using zoom-`ticks` instead, before I try to make any suggestions here.